src/client/components/MobileMsg.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+
+import lightScreen from '@resources/assets/screenshot-light.png'
+import darkScreen from '@resources/assets/screenshot-dark.png'
+import squareLogo from '@resources/assets/logo-square-white.svg'
+import logo from '@resources/assets/logo-square-color.svg'
+
+export const MobileMsg: React.FC = () => {
```

Comment:
This component contains a lot of duplicate code from LandingPage. If we want to update the landing page we would have to now update this component too. It seems the 'View demo' button just needs to conditionally display based on device type.

Reply:
I'm very confused @kodyclemens, so what would be showing up for tablet and mobile devices?

Reply:
Why not extend the conditional linked below to include an `isMobile` check? There is no need for MobileMsg, which is basically a duplicate landing page for mobile devices.

[Conditional](https://github.com/taniarascia/takenote/blob/174c1ad2507b213f5d31238ce91e52ada01b15ad/src/client/components/LandingPage.tsx#L36-L55)

Reply:
Done @kodyclemens. Btw, `import { Link } from 'react-router-dom'` is declared but its value is never read. We remove it?

Reply:
Yeah I would remove unused imports